pkg/controller/installation/products/codeready/reconciler.go

```diff
@@ -454,8 +464,27 @@ func (r *Reconciler) getBackupConfig() resources.BackupConfig {
 	}
 }
 
-func (r *Reconciler) createCheCluster(ctx context.Context, kcCfg *config.RHSSO, kr *keycloakv1.KeycloakRealm, inst *v1alpha1.Installation, serverClient pkgclient.Client) error {
+func (r *Reconciler) createCheCluster(ctx context.Context, kcCfg *config.RHSSO, kr *keycloakv1.KeycloakRealm, inst *v1alpha1.Installation, serverClient pkgclient.Client) (*chev1.CheCluster, error) {
 	selfSignedCerts := inst.Spec.SelfSignedCerts
+	cheDb := chev1.CheClusterSpecDB{
+		ExternalDB:            false,
+		ChePostgresDb:         "",
+		ChePostgresPassword:   "",
+		ChePostgresPort:       "",
+		ChePostgresUser:       "",
+		ChePostgresDBHostname: "",
+	}
+	// setup external postgres db if UseExternalResource set to true
+	if inst.Spec.UseExternalResources {
```

Comment:
We should remove this `inst.Spec.UseExternalResources` so it provisions an in-cluster postgres by default

Reply:
have we got rid of UseExternalResources in the installation cr ?

Reply:
Yep

Reply:
It's not merged yet though

Reply:
so we are using CRO to install openshift resources, all resources are by default external no matter where they are hosted from now on.